src/Data/Text/Utf8/AhoCorasick/Automaton.hs

```diff
@@ -380,11 +388,15 @@ buildValueMap transitions fallbacks valuesInitial =
 -- in one place. We ran this code with `Vector.!` (bounds-checked indexing) in
 -- production for two months without failing the bounds check, so we have turned
 -- the check off for performance now.
+{-# INLINE at #-}
 at :: forall a. Vector.Vector a -> Int -> a
 at = Vector.unsafeIndex
 
-uAt :: forall a. UVector.Unbox a => UVector.Vector a -> Int -> a
-uAt = UVector.unsafeIndex
+{-# INLINE uAt #-}
+{-# SPECIALIZE INLINE uAt :: TypedByteArray Offset -> Int -> Offset  #-}
+{-# SPECIALIZE INLINE uAt :: TypedByteArray Transition -> Int -> Transition  #-}
```

Comment:
These shouldn't be necessary if we have a more generic `INLINE` pragma already, right? The docs say "The INLINE pragma affects the specialised version of the function (only)", so weaker than a normal `INLINE` (unless you have a type-recursive function, where it can make sense to have).

Reply:
Good catch, those were not meant to still be here :sweat_smile: